src/onnx/onnx.cpp

```diff
@@ -435,14 +456,54 @@ struct onnx_parser
         {
             copy(attributes["kernel_shape"].ints(), op.lengths.begin());
         }
+
         if(contains(attributes, "auto_pad"))
```

Comment:
can we create a separate function for auto_pad? We can use it for conv and conv_transpose as well